lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
```

Comment:
They avoid it in good Scala code. (But most Scala code isn't good.) Haskell makes you jump through hoops if you want mutation.
That said, some algorithms and data structures cannot be implemented efficiently without mutation.